### DISCUSSION TITLE
I can't apply to the teacher program using the email address with the domain apd.edu.vn, as the system requires st.apd.edu.vn.

### DISCUSSION THREAD

**Discussion by phantuanngoc68** [community/community#146480] [GitHub Education]
### Hi everyone,

 I can't apply to the teacher program using the email address with the domain apd.edu.vn, as the system requires st.apd.edu.vn. However, the organization only provides teachers with the apd.edu.vn domain. What should I do?

**Comment by davevad93** [comment]
Hi @phantuanngoc68 ,

Try contacting support by opening a ticket [here](https://support.github.com/contact/education), select "`My selected school has incorrect or incomplete information e.g. domains or campus location`" and explain your problem.

**Reply by phantuanngoc68** [reply]
Thank you so much for taking the time to help me with this issue! 😊 I’ve already followed your suggestion and submitted the necessary updates. Now, I’m waiting for a response

**Reply by davevad93** [reply]
You're welcome @phantuanngoc68 , I hope it gets fixed. 👍
